sktime/transformations/series/fourier.py

```diff
@@ -45,12 +45,16 @@ class FourierFeatures(BaseTransformer):
     ----------
     sp_list : List[float and/or str]
         List of seasonal periods. Can be defined with the following options:
+
         * float : Periodicity defined as number of timesteps since the beginning of the
-        data seen in `fit`.
+        data seen in ``fit``.
```

Comment:
This is being displayed in a new line, which looks bad. I think adding an indentation may solve this, but I'm not certain.

```suggestion
            data seen in ``fit``.
```

Reply:
I'll make the changes and see if it works. Is it possible to build just the FourierFeatures reference page locally?

Reply:
Not just one page, no. You can absolutely build `sktime` docs locally, but with default settings and files it may take 10-15 minutes. If you're okay to make some temporary changes (commenting few lines here and there and so on, remove few files temporarily), it will be much first, but you have to remember to restore all those. I'll not recommend that if you're just starting with Sphinx, it may be a bit too complex.